apps/keck/src/server/sync/collaboration.rs

```diff
@@ -130,14 +135,17 @@ mod test {
 
         fs::remove_dir_all("./data").unwrap();
         close_collaboration_server(child);
+
+        // workaround to disable parallel running with following test, which will
+        // cause memory sqlite db data incorrect
+        client_collaboration_with_server_with_poor_connection();
     }
 
-    #[test]
-    #[ignore="client_collaboration_with_server cannot close websocket gracefully, causing this fails"]
+    // #[test]
     fn client_collaboration_with_server_with_poor_connection() {
         create_db_dir();
-        jwst_logger::init_logger();
-        let child = start_collaboration_server();
+        let server_port = 65535;
```

Comment:
is recommend use a random number

Reply:
Okay, I'll deal with it later. Do we need two different ranges? Otherwise they may conflict

Reply:
generally we can choose in 10000 > n > 65535

Reply:
11 use rand as server port now